Dockerfile

```diff
@@ -1,23 +1,62 @@
-FROM registry.access.redhat.com/ubi9/python-311
-ARG POETRY_VERSION
-RUN pip install --upgrade pip && \
-    pip install poetry==$POETRY_VERSION
+FROM quay.io/app-sre/er-base-cdktf-aws:0.2.1 AS prod
+COPY --from=ghcr.io/astral-sh/uv:0.4.26@sha256:7775c60dca9cc5827c36757c32c75985244d8f31447565fa8147e2b2e11ad280 /uv /bin/uv
```

Comment:
Is `uv` needed at the final prod image? Can we follow uv multistage example https://github.com/astral-sh/uv-docker-example/blob/main/multistage.Dockerfile to have builder, test and prod.

Also use `--frozen` when install deps to avoid lockfile change during build.